docs/source/python-publish.yml

```diff
@@ -0,0 +1,33 @@
+# This workflows will upload a Python Package using flit when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
```

Comment:
Can we rather name that file as `publish-pypi.yml` or `publish-python.yml` to group the relevant workflows?